### PR TITLE
Remove IGListStackedSectionController

### PR DESCRIPTION
**Pull request by zhubofei**
## Changes in this pull request

Issue fixed: #1099

### Checklist

- [x] All tests pass. Demo project builds and runs.
- [x] I added tests, an experiment, or detailed why my change isn't tested.
- [x] I added an entry to the `CHANGELOG.md` for any breaking changes, enhancements, or bug fixes.
- [x] I have reviewed the [contributing guide](https://github.com/Instagram/IGListKit/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by zhubofei** [comment]
Deprecating IGListStackedSectionController means removing it completely, right? Or just add `__deprecated` to the class? Should have asked in advance 😩

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,9 @@ The changelog for `IGListKit`. Also see the [releases](https://github.com/instag
 -----
 ### Breaking Changes
 
-- All `IGListBindingSectionControllerSelectionDelegate` methods are now required. [Bofei Zhu] (https://github.com/zhubofei) [(#1186)](https://github.com/Instagram/IGListKit/pull/1186)
+- All `IGListBindingSectionControllerSelectionDelegate` methods are now required. [Bofei Zhu](https://github.com/zhubofei) [(#1186)](https://github.com/Instagram/IGListKit/pull/1186)
+
+- `IGListStackedSectionController` has been completely removed. [Bofei Zhu](https://github.com/zhubofei) (tbd)
```
Comment (jessesquires) [inline CHANGELOG.md]: can fill-in this `tbd` with this PR link 😄

**Comment by rnystrom** [comment]
We kind-of-not-really use this once within Instagram. I'll accept this but it'll take me some time to import and fixup internal build errors. Thank you so much @zhubofei for driving this!!

**Comment by lorixx** [comment]
@bofeizhu  Hi Bofei, can you rebase to master again? thanks!

**Comment by bofeizhu** [comment]
@lorixx Sorry, won’t be able to work on this for the coming months. Also, I think it’s easier to open a new PR at this point. I’ll go ahead and close this PR. Feel free to create a new PR for this issue! Thx!

**Comment by hanton** [comment]
hi @bofeizhu @lorixx , I would like to help creating the new PR #1355 :)

It's based on the latest master branch, so the PR has no merge conflicts and the commits are more clean.

**Comment by bofeizhu** [comment]
@hanton Thank you!

**Comment by Ziewvater** [comment]
I think this was handled by [1099](https://github.com/Instagram/IGListKit/issues/1099), so closing

**Pull request closed without merging**